README.md

```diff
@@ -1,50 +1,49 @@
-# About
-A web application for bicycle collectives to track bicycles, bicycle work history, volunteer hours, volunteer work history, and volunteers currently in the shop.
+﻿# About
 
-See a live demo here: <http://bikeshed.wvcompletestreets.org/>  Thanks @fspc for setting that up!
+  A web application for bicycle collectives to track bicycles, bicycle work history, volunteer hours, volunteer work history, and volunteers currently in the shop.
 
-See an overview video of what the desktop view looks like and how it works here: https://www.youtube.com/watch?v=0-JjM6d9nK4.
+  See a live demo here: http://bikeshed.wvcompletestreets.org/ Thanks @fspc for setting that up!
 
-## Overview/Instructions/Guides
+  See an overview video of what the desktop view looks like and how it works here: https://www.youtube.com/watch?v=0-JjM6d9nK4.
 
-See [guides](doc/guides.md)
+  Overview/Instructions/Guides
 
-# Developer Setup
+  See guides
 
-## Running with Docker (recommended)
+#  Developer Setup
 
-1. These instructions haven't been tested, please provide corrections!
```

Comment:
These numbers were necessary, now it looks like this which makes it difficult to read the different steps:

<img width="1171" alt="screen shot 2016-07-27 at 2 01 26 pm" src="https://cloud.githubusercontent.com/assets/384389/17192306/b83155f0-5402-11e6-8428-6205f050e3f3.png">